### DISCUSSION TITLE
Entity e Logs

### DISCUSSION THREAD

**Discussion by fabioborgesconttrotech** [balta-io/forum#498] [Tira Dúvidas] [answered]
Balta,
No curso de fundamentos do Entity (Introdução > Update) você menciona que o Entity trabalha com metadados e consegue ter informações sobre o registro como ultimas modificações e tal.... nesse sentido tem algo no Entity que auxilia a geração de logs.... como por exemplo registrar um campo que foi alterado (de para)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @fabioborgesconttrotech, tudo bem por aí?

Auxilia sim, dá uma olhada nesta funcionalidade:
https://learn.microsoft.com/pt-br/ef/core/logging-events-diagnostics/

**Comment by fabioborgesconttrotech** [comment]
Obrigado Balta....

 

Fabio Borges

Para: balta-io/forum ***@***.***>
Cc: Fabio Borges ***@***.***>; Mention ***@***.***>
Assunto: Re: [balta-io/forum] Entity e Logs (Discussion #498)

 

Bom dia, @fabioborgesconttrotech <https://github.com/fabioborgesconttrotech> , tudo bem por aí?

Auxilia sim, dá uma olhada nesta funcionalidade:
https://learn.microsoft.com/pt-br/ef/core/logging-events-diagnostics/
